codegenerator/cli/templates/static/codegen/src/globalState/GlobalState.res

```diff
@@ -576,7 +576,7 @@ let actionReducer = (state: t, action: action) => {
     if isInReorgThreshold {
       Logging.info("Reorg threshold reached")
     }
-    ({...state, chainManager: {...state.chainManager, isInReorgThreshold}}, [ProcessEventBatch])
```

Comment:
Is it sure that we never get into a deadlock where ProcessEventBatch doesn't get dispatched again?

Reply:
Yep